test/defaults.test.js

```diff
@@ -11,15 +11,15 @@ describe('Defaults', function () {
     });
     r('http://www.filltext.com/?rows=1', function (err, response, body) {
       t.strictEqual(response.statusCode, 200);
-      t.strictEqual(body[0].d, 1);
+      t.isNumber(body[0].d);
```

Comment:
Hum, why not. I prefer to ensure that the response is as it should be (because it's free here to do so), why would we want to prefer isNumber here?